test/testUtils/quickBuilderUtil.js

```diff
@@ -2,9 +2,9 @@ const nodeUtils = require("../../src/common/utils");
 const EnviornmentBuilder = require("../../src/main_controller/EnvironmentBuilder");
 const CoreServer = require("../../src/core/core_server_mock/core_server");
 const path = require("path");
-const _B1Path = path.join(__dirname, "./id-l");
+const _B1Path = path.join(__dirname, "./id-l.json");
```

Comment:
Also 2, remove the weird `./` and split `"./testUtils/id-l.json"` to `"testUtils", "id-l.json"`.